src/commands/scanner/run/dfa.ts

```diff
@@ -43,29 +42,30 @@ export default class Dfa extends ScannerRunCommand {
 			summary: getMessage(BundleName.RunDfa, 'flags.targetSummary'),
 			description: getMessage(BundleName.RunDfa, 'flags.targetDescription'),
 			delimiter: ',',
+			default: '.',
 			multiple: true
 		})(),
 		// END: Flags for targeting files.
 		// BEGIN: Config-overrideable engine flags.
 		'rule-thread-count': Flags.integer({
 			summary: getMessage(BundleName.RunDfa, 'flags.rulethreadcountSummary'),
-			description: getMessage(BundleName.RunDfa, 'flags.rulethreadcountDescription'),
+			default: 4,
 			env: 'SFGE_RULE_THREAD_COUNT'
 		}),
 		'rule-thread-timeout': Flags.integer({
 			summary: getMessage(BundleName.RunDfa, 'flags.rulethreadtimeoutSummary'),
-			description: getMessage(BundleName.RunDfa, 'flags.rulethreadtimeoutDescription'),
+			default: 900000,
 			env: 'SFGE_RULE_THREAD_TIMEOUT'
 		}),
 		// NOTE: This flag can't use the `env` property to inherit a value automatically, because OCLIF boolean flags
 		// don't support that. Instead, we check the env-var manually in a subsequent method.
 		'rule-disable-warning-violation': Flags.boolean({
 			summary: getMessage(BundleName.RunDfa, 'flags.ruledisablewarningviolationSummary'),
-			description: getMessage(BundleName.RunDfa, 'flags.ruledisablewarningviolationDescription')
+			description: getMessage(BundleName.RunDfa, 'flags.ruledisablewarningviolationDescription'),
+			default: 'false'
```

Comment:
I don't think we want a string false value... but a boolean false value. But in either case, I need to check if setting this here is safe or not based on how we use it in our code.